export function handleError(error) {
  console.log('Error: ', error);
}
